acceptance/localcluster/localcluster.go

```diff
@@ -332,6 +332,7 @@ func (l *Cluster) processEvent(e dockerclient.EventOrError) bool {
 	// An event on any other container is unexpected. Die.
 	select {
 	case <-l.stopper:
+	case <-monitorStopper:
```

Comment:
The problem this change fixes is that Stop() may be called while the `monitor()` goroutine is still running. This clears `l.Nodes`, causing any remaining events to be treated as "unexpected". With this change, any late-arriving events will be silently discarded.

The rest of the thread deals in hypotheticals: `processEvents` can't use `l.monitorStopper` because that field is set to nil in `Stop`, and we must continue to clear `l.monitorStopper` in `Stop` or else there would be a panic when `Stop` is called twice.